.changeset/thirty-paws-rush.md

```diff
@@ -0,0 +1,16 @@
+---
+'@shopify/ui-extensions': minor
+---
+
+- Added customer account migration guide with comparison tables mapping legacy components to new Polaris web components
```

Comment:
I don't think we need to include docs in the changesets 🤔 It won't hurt, though.